src/app/shared/models/truncated.model.ts

```diff
@@ -0,0 +1,14 @@
+export class RequestParams{
+  id: string;
+  isParent: boolean;
+
+  constructor(id: string, isParent: boolean){
+    this.id = id;
+    this.isParent = isParent;
+  }
+}
```

Comment:
don't think we need a constructor here, let's stay with interface, and probably it should be moved to the children ts file